web/src/compose_banner.ts

```diff
@@ -131,7 +131,14 @@ export function show_error_message(
     classname: string,
     $container: JQuery,
     $bad_input?: JQuery,
+    html_content: boolean = false,
```

Comment:
This option intentionally does not exist; you're supposed to use partials like `web/templates/compose_banner/automatic_new_visibility_policy_banner.hbs`

I pushed a prep commit adding a comment explaining this.